recursion/problem-set-recursion-pt-1.md

```diff
@@ -210,6 +210,7 @@ Write a function `is_nested_parens` that accepts a string `parens` of only paren
 | `"((()))"`      | `True`          |
 | `""`            | `True`          |
 | `"(())))"`      | `False`         |
+| `"(()())"`      | `False`         |
```

Comment:
As mentioned in the Slack thread, I think having this example may be weird without a definition of "properly nested" parentheses. Students may have different ideas of what a "properly nested" parenthesis is given the fact that this suggestion came from a student themself. I think we may want to clarify either before or after the examples are given. I've left a suggestion below:

```suggestion
| `"(()())"`      | `False`         |

*Note: Parentheses are considered nested if and only if you can divide the string in two and the right side only consists of opening parentheses and the left side only consists of closing parentheses. As a result, the final example here fails.
```